src/api/index.ts

```diff
@@ -1,7 +1,9 @@
 import express from 'express'
 import adminRouter from './admin'
+import { services } from '../services'
 
 const apiRouter = express.Router()
 apiRouter.use('/admin', adminRouter.app)
+apiRouter.use('/services', services)
```

Comment:
1. I think `providers` is a better name (sorry!)
2. This is an API, should be appropriately prefixed: `/api/providers`

Reply:
To get all providers domains: `/api/providers/domains`
To get domains for a specific provider :/api/providers/domains/godaddy